doc/guides/encoding/jpeg_2k.rst

```diff
@@ -0,0 +1,291 @@
+encoding. For an explanation of each parameter and its relationship
+with the *Pixel Data* see the :doc:`glossary of Image Pixel elements<../glossary>`.
+
++------------+-----------------------+-----------------+----------------+------------+---------+
+| *Samples   | *Photometric          | *Pixel          | *Planar        | *Bits      | *Bits   |
+| per Pixel* | Interpretation*       | Representation* | Configuration* | Allocated* | Stored* |
++============+=======================+=================+================+============+=========+
+| 1          | MONOCHROME1           | 0 or 1          | (absent)       | 8, 16, 24, | 1 to 38 |
+|            +-----------------------+                 |                | 32 or 40   |         |
+|            | MONOCHROME2           |                 |                |            |         |
+|            +-----------------------+-----------------+----------------+------------+---------+
+|            | PALETTE COLOR :sup:`1`| 0               | (absent)       | 8 or 16    | 1 to 16 |
++------------+-----------------------+-----------------+----------------+------------+---------+
+| 3          | RGB                   | 0               | 0              | 8, 16, 24, | 1 to 38 |
+|            +-----------------------+                 |                | 32 or 40   |         |
+|            | YBR_RCT               |                 |                |            |         |
+|            +-----------------------+                 |                |            |         |
+|            | YBR_ICT :sup:`2`      |                 |                |            |         |
+|            +-----------------------+                 |                |            |         |
+|            | YBR_FULL              |                 |                |            |         |
++------------+-----------------------+-----------------+----------------+------------+---------+
+
+| :sup:`1` *JPEG 2000 Lossless* only
+| :sup:`2` *JPEG 2000* only
+
+Photometric Interpretation
+..........................
+
+To ensure you have the correct *Photometric Interpretation* the uncompressed
+pixel data should already be in the corresponding color space:
+
+* If your uncompressed pixel data is grayscale (intensity) based:
+
+  * Use ``MONOCHROME1`` if the minimum intensity value should be displayed as
+    white.
+  * Use ``MONOCHROME2`` if the minimum intensity value should be displayed as
+    black.
+
+* If your uncompressed pixel data uses a single sample per pixel and is an index
+  to the :dcm:`Red, Green and Blue Palette Color Lookup Tables
+  <part03/sect_C.7.6.3.html#sect_C.7.6.3.1.5>`:
+
+  * Use ``PALETTE COLOR``.
+
+* If your uncompressed pixel data is in RGB color space:
+
+  * For *Photometric Interpretation* ``RGB``, ``YBR_ICT`` and ``YBR_RCT`` nothing
+    else is required.
+  * For *Photometric Interpretation* ``YBR_FULL`` the pixel data must first be
```

Comment:
I suggest 'or' here rather than 'and' and a colon after the ``YBR_RCT`` would read better; I first read the part after the first comma as something applying to the case of ``RGB``.

If you agree, then colons for the other cases too...

Reply:
Hopefully I've changed what you meant, although I only added a semi-colon where there were multiple PIs